core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -222,6 +225,13 @@ export class RteHelperModalComponent {
 
     this.customizationArgsForm = this.fb.group(formGroupControls);
 
+    if (this.componentId === this.COMPONENT_ID_MATH ||
+      this.componentId === this.COMPONENT_ID_VIDEO ||
+      this.componentId === this.COMPONENT_ID_LINK
+    ) {
+      this.subscribeToRteFieldChanges();
```

Comment:
Why not always subscribe, regardless of the component type? Then you don't need "if this.rteFieldValues" everywhere.

Reply:
done